### ISSUE TITLE
Blocked on "Please Wait..." after the 1.5.0 update

### ISSUE THREAD

**Issue by scrum-master-org** [GoogleForCreators/web-stories-wp#6856]
## Bug Description
Hello,

I'm facing an issue on my wordpress Website.
Few days ago I successfully created 2 web-stories.
but since yesterday (after the update the plugin to 1.5.0 version and wordpress to 5.7 version) I'm having an empty black or white screen when  want to see or create a new story. 
I only see the text : "Please Wait..."

## Expected Behaviour

Be able to create a new story in my Wordpress site.

## Steps to Reproduce

update t o 1.5.0 and the tr to create a Story with the plugin or go on : "My Stories" Page

## Screenshots

[Link of Screnshot 1](https://scrum-master.org/wp-content/uploads/2021/03/web-story-google-bug-wordpress-please-wait.png)
[Link of Screenshot 2 ](https://scrum-master.org/wp-content/uploads/2021/03/web-story-google-bug-wordpress-please-wait.png)

You can see here my two previous stories that I successfully created before I updated the plugin :
[Working previous web stories](https://scrum-master.org/web-stories/)

## Additional Context

- Plugin Version: 1.5.0
- WordPress Version: 5.7
- Operating System: Windows 10
- Browser: Chrome 89

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by LuckynaSan** [comment]
@scrum-master-org Sorry to hear you‘re having issues. Since we’ve moved support to the WordPress.org support forums, could you please open a new support topic at https://wordpress.org/support/plugin/web-stories/? 

To try and determine the cause, please include as much information as possible, like WP version, browser version, console errors, Site Health data, etc. Looking forward to resolving this matter with you there. Thank you.

**Comment by LuckynaSan** [comment]
@scrum-master-org Is this issue still occurring for you? Could you share who your hosting provider is and do you have any server-side caching or security measures enabled? If yes, could you temporarily disable it to rule out if this could be involved?

**Comment by scrum-master-org** [comment]
Hello the issue is still here ! I'm still blocked on an empty screen with the message "Please Wait..."

Can you contact me via email to arrange an access with admin rights so you can check further this issue?

Sincerely, Ahmed

**Comment by LuckynaSan** [comment]
While we are not allowed admin access per these [guidelines](https://wordpress.org/support/guidelines/#the-bad-stuff), we’d like to work with you to get Web Stories up and running again. 

If possible, could you perhaps try installing this in-development version? Download:
[web-stories.zip](https://github.com/google/web-stories-wp/files/6205680/web-stories.zip)

If you are still experiencing the issue, could you please share your [Site Health](https://i.imgur.com/Rtkmsba.png) information and the below details?
Who is your hosting provider? 
Do you have any server-side caching or security measures enabled?

**Comment by scrum-master-org** [comment]
hello, 
I'm hosted in Hostinger.
this is my site health data :
`
### wp-core ###

version: 5.7
site_language: fr_FR
user_language: fr_FR
timezone: Europe/Paris
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 1
blog_public: 1
default_comment_status: open
environment_type: production
user_count: 4
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /home/u188096912/domains/scrum-master.org/public_html
wordpress_size: 59,48 Mo (62366585 bytes)
uploads_path: /home/u188096912/domains/scrum-master.org/public_html/wp-content/uploads
uploads_size: 120,06 Mo (125888938 bytes)
themes_path: /home/u188096912/domains/scrum-master.org/public_html/wp-content/themes
themes_size: 18,29 Mo (19181613 bytes)
plugins_path: /home/u188096912/domains/scrum-master.org/public_html/wp-content/plugins
plugins_size: 102,39 Mo (107358729 bytes)
database_size: 53,80 Mo (56410112 bytes)
total_size: 354,01 Mo (371205977 bytes)

### wp-active-theme ###

name: Astra (astra)
version: 3.2.0
author: Brainstorm Force
author_website: https://wpastra.com/about/
parent_theme: none
theme_features: core-block-patterns, astra_hooks, align-wide, automatic-feed-links, title-tag, post-thumbnails, html5, post-formats, custom-logo, customize-selective-refresh-widgets, editor-style, woocommerce, amp, infinite-scroll, web-stories, widgets, menus
theme_path: /home/u188096912/domains/scrum-master.org/public_html/wp-content/themes/astra
auto_update: Désactivé

### wp-plugins-active (24) ###

Akismet Anti-Spam: version: 4.1.9, author: Automattic, Mises à jour auto désactivées
All In One WP Security: version: 4.4.8, author: Tips and Tricks HQ, Peter Petreski, Ruhul, Ivy, Mises à jour auto désactivées
Antispam Bee: version: 2.9.4, author: pluginkollektiv, Mises à jour auto désactivées
Archive Title by WebMan: version: 1.0.0, author: WebMan Design, Oliver Juhas, Mises à jour auto désactivées
Astra Pro: version: 3.2.0, author: Brainstorm Force, Mises à jour auto désactivées
Astra Widgets: version: 1.2.8, author: Brainstorm Force, Mises à jour auto désactivées
bbPress: version: 2.6.6, author: The bbPress Contributors, Mises à jour auto désactivées
bbp style pack: version: 4.8.6, author: Robin Wilson, Mises à jour auto désactivées
Classic Editor: version: 1.6, author: WordPress Contributors, Mises à jour auto désactivées
Code Snippets: version: 2.14.1, author: Code Snippets Pro, Mises à jour auto désactivées
Contact Form 7: version: 5.4, author: Takayuki Miyoshi, Mises à jour auto désactivées
Elementor: version: 3.1.4, author: Elementor.com, Mises à jour auto désactivées
Elementor - Header, Footer & Blocks: version: 1.5.7, author: Brainstorm Force, Nikhil Chavan, Mises à jour auto désactivées
Feedzy RSS Feeds Lite: version: 3.5.2, author: Themeisle, Mises à jour auto désactivées
Import / Export Customizer Settings: version: 1.0.4, author: Brainstorm Force, Mises à jour auto désactivées
Jetpack by WordPress.com: version: 9.5, author: Automattic, Mises à jour auto désactivées
LiteSpeed Cache: version: 3.6.4, author: LiteSpeed Technologies, Mises à jour auto désactivées
Rank Math SEO: version: 1.0.60.1, author: Rank Math, Mises à jour auto activées
Smush: version: 3.8.3, author: WPMU DEV, Mises à jour auto désactivées
Starter Templates: version: 2.6.2, author: Brainstorm Force, Mises à jour auto désactivées
Table of Contents Plus: version: 2002, author: Michael Tran, Mises à jour auto désactivées
Web Stories: version: 1.5.0, author: Google, Mises à jour auto désactivées
White Label CMS: version: 2.2.2, author: www.videousermanuals.com, Mises à jour auto désactivées
Yoast Duplicate Post: version: 4.1.2, author: Enrico Battocchi & Team Yoast, Mises à jour auto désactivées

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1802
imagemagick_version: ImageMagick 7.0.10-10 Q16 x86_64 2020-07-09 https://imagemagick.org
file_uploads: File uploads is turned off
post_max_size: 512M
upload_max_filesize: 512M
max_effective_size: 512 Mo
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 752 Go
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 37500
	imagick::RESOURCETYPE_MAP: 752 Go
	imagick::RESOURCETYPE_MEMORY: 376 Go
	imagick::RESOURCETYPE_THREAD: 1
gd_version: 2.2.5
ghostscript_version: 9.25

### wp-server ###

server_architecture: Linux 3.10.0-962.3.2.lve1.5.42.el7.x86_64 x86_64
httpd_software: LiteSpeed
php_version: 7.4.13 64bit
php_sapi: litespeed
max_input_variables: 5000
time_limit: 300
memory_limit: 1024M
max_input_time: 240
upload_max_filesize: 512M
php_post_max_size: 512M
curl_version: 7.71.0 OpenSSL/1.1.1d
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.3.25-MariaDB-cll-lve
client_version: mysqlnd 7.4.13

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/u188096912/domains/scrum-master.org/public_html/wp-content
WP_PLUGIN_DIR: /home/u188096912/domains/scrum-master.org/public_html/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 1024M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### jetpack ###

site_id: 189660458
ssl_cert: No
time_diff: undefined
version_option: 9.5:1614713647
old_version: 9.4:1612912799
public: Public
master_user: #4 Ahmed BEN SALEM
current_user: #4 Ahmed BEN SALEM
tokens_set: Blog User
blog_token: vHLsVMvsNpUw8ZUzqT!Lb!M66xxtcTBs
user_token: pyllrzgt2ErxbK#iYvb1!j9axjFh@)LY
version: 9.5
jp_plugin_dir: /home/u188096912/domains/scrum-master.org/public_html/wp-content/plugins/jetpack/
plan: free
protect_header: {"trusted_header":"REMOTE_ADDR","segments":1,"reverse":false}
full_sync: {"started":"Sun, 21 Feb 2021 20:45:22 +0000","finished":"Sun, 21 Feb 2021 20:45:26 +0000","progress":{"options":{"finished":true},"functions":{"finished":true},"constants":{"finished":true},"users":{"total":"1","sent":1,"finished":true,"last_sent":"4"},"network_options":{"finished":true}},"config":{"options":true,"functions":true,"constants":true,"users":[4],"network_options":true}}
sync_size: undefined
sync_lag: 0 seconds
full_sync_size: undefined
full_sync_lag: 0 seconds
idc_urls: {"home":"https:\/\/scrum-master.org","siteurl":"https:\/\/scrum-master.org","WP_HOME":"","WP_SITEURL":""}
idc_error_option: false
idc_optin: true
cxn_tests: All Pass.

`

**Comment by scrum-master-org** [comment]
I'm still facing the same issue with the alpha version that you just sent me.

Sincerely, Ahmed

**Comment by LuckynaSan** [comment]
@scrum-master-org Thank you for the updates! 

1) Are you using Cloudflare CDN services on Hostinger?

2) To confirm, did this issue occur with the development version and in an [incognito](https://i.imgur.com/ALzgiHp.png) window? 

3) If you open your browser developer tools, are you seeing any [browser console errors](https://imgur.com/KyLAzgH.png) where the "Please wait..." message displays?

4) To rule out any conflicting plugin/theme, could you please activate the <a href="https://wordpress.org/plugins/health-check/">Health Check &amp; Troubleshooting plugin</a> and enable its <a href="https://make.wordpress.org/support/handbook/appendix/troubleshooting-using-the-health-check/#troubleshooting">troubleshooting mode</a>. This mode disables active plugins and switches to a default theme safely because it only affects the experience of the logged-in user. 

- Please enable the <a href="https://jmp.sh/dw3iCRp">Web Stories plugin</a> and let me know if the issue is present.
- Enabling plugins and your theme one by one in this mode may help pinpoint if something else is involved.
- Note you can <a href="https://jmp.sh/uErQi8t">disable troubleshooting mode</a> from the plugins page.

**Comment by scrum-master-org** [comment]
Hello : 

1. Yes I use Cloudflare as a CDN.
2. the errors still occur with the development version in incognito mode.
3. I still have the same console errors as I sent here in a previous message. 
4. I enabled the troubleshooting mode and with only Google web-Stories Activated and all the other plusgins deactivated I'm still facing the same issue. the console error message is : vendors~edit-story~stories-dashboard-e67f594209a70c11360e.js:1 Failed to load resource: the server responded with a status of 403 ()

sincerely, Ahmed

**Comment by scrum-master-org** [comment]
I also just right now did activated the Cloudflare "development mode" to bypass cloudflare cache. And the error is still here :(

**Comment by LuckynaSan** [comment]
A 403 error could mean there is a firewall block or that access to the web page is not allowed for some reason. Do you have any firewall setting on Cloudflare that blocks any URL such as WP-JSON or the like? 

Where you see that error, it would be great if we can get the _Response_ on the request too. Here's how to do it:

- In the dev tools, go to the _Network tab_
- Find and select the request that failed under the _Name column_. It should be highlighted in red (you may need to refresh the page again).
- Go to the _Response tab_, and take a screenshot or copy/ paste what is there. 

Here's a video example on where to navigate: https://recordit.co/90feR8mX5h

Thanks!

**Comment by LuckynaSan** [comment]
@scrum-master-org This issue has been resolved in the newest version of the Web Stories for WordPress plugin. Please update to version 1.6.0 of the plugin and reach out in the [WordPress.org support forums](https://wordpress.org/support/plugin/web-stories/) if you continue to experience any issues. Thank you!
